mlir/lib/Target/LLVM/ModuleToObject.cpp

```diff
@@ -215,6 +222,10 @@ std::optional<SmallVector<char, 0>> ModuleToObject::run() {
   }
   setDataLayoutAndTriple(*llvmModule);
 
+  if (initialLlvmIRCallback) {
+    initialLlvmIRCallback(*llvmModule);
+  }
```

Comment:
```suggestion
  if (initialLlvmIRCallback)
    initialLlvmIRCallback(*llvmModule);
```

The same applies to other cases.

Reply:
Fixed, thanks!